python2/typing.py

```diff
@@ -1801,7 +1801,8 @@ def NamedTuple(typename, fields):
     """
     fields = [(n, t) for n, t in fields]
     cls = collections.namedtuple(typename, [n for n, t in fields])
-    cls._field_types = dict(fields)
+    cls.__annotations__ = collections.OrderedDict(fields)
```

Comment:
``get_type_hints`` always returns ``None`` in Python 2. So that I would prefer to leave this unchanged.